src/installation/install_epinio_on_rancher.md

```diff
@@ -0,0 +1,57 @@
+# Rancher configuration
+
+Before installing Epinio, you need a running Rancher instance.
+
+If you don't get it installed yet, feel free to follow installation guides [here](https://rancher.com/docs).
+
+## Epinio Prerequisites
+
+### Kubernetes cluster
+For development purpose, you can deploy Epinio on the same node where Rancher installed but it's not supported for production.</br>
+That's why first, we are about to detail how to install Epinio in a separate Kubernetes cluster.</br>
+In this case, you need to either deploy a new cluster or import an existing one in Rancher. Those steps are detailed in the Rancher docs.
+
+### Rancher settings
+At the time of this documentation is written, Epinio is not included in the latest stable Rancher version.</br>
+That means you need to update two settings in your Rancher dashboard.</br>
```

Comment:
MAybe better to use `This` instead of `That`? I checked [here](https://keydifferences.com/difference-between-this-and-that.html#:~:text=The%20words%20'this'%20and%20',is%20used%20for%20indicating%20something.&text=We%20use%20the%20word%20'this,which%20is%20farther%20from%20you.) but I'm still confused :D.